app/search_builders/hyrax/filter_by_type.rb

```diff
@@ -30,7 +30,9 @@ def models
     end
 
     def models_to_solr_clause
-      models.map { |model| model.try(:to_rdf_representation) || model.name }.join(',')
+      models.map do |model|
+        model.respond_to?(:to_rdf_representation) ? model.to_rdf_representation : model.name
```

Comment:
some of our classes now have a `.try` method from `Dry::Struct` which behaves differently. since we're no longer stubbing this behavior out, we need to fix it.